evil-collection-help.el

```diff
@@ -51,6 +51,8 @@
     ;; "\C-j" 'help-go-forward
     ;; "\C-k" 'help-go-back
 
+    "go" 'push-button ;; this should open in same window
+    "gO" 'push-button ;; this should open in another window
```

Comment:
Are these comments true? These both trigger 'push-button'.

Reply:
thats why I put should.  Neither do what they should `push-button` sometimes opens in the same window sometimes in other-window context-wise.

Reply:
OK, it might be better to clarify the comments here.

Reply:
Ok, do you have a suggestion on implementation? I don't see the point on writing two functions here, I would just bind both as in the commit.

Reply:
You can just update the comments here to reflect that they 'should' open in same/another window *but* don't in all cases.

Reply:
Ok, that I will do.